Tutorial-Learn-How-to-Make-a-Simple-UI-Change.md

```diff
@@ -1,5 +1,3 @@
-# Tutorial: Making a Simple UI Change
```

Comment:
You've deleted the header of the page, so what is the page going to look like on the GitHub wiki? Will readers know what the page refers to?

Reply:
Github wiki automatically renders the file name as the top level header (I learnt about it just recently) - https://github.com/oppia/oppia/wiki/Making-a-simple-UI-change. If you checkout the wiki page, you could see two headers which is making it look a bit odd.